lib/neoboard/widgets/mattermost.ex

```diff
@@ -3,34 +3,55 @@ defmodule Neoboard.Widgets.Mattermost do
   use Neoboard.Pusher
   use Neoboard.Config
   alias Neoboard.TimeService
+  alias Neoboard.Widgets.Mattermost.Authentication
+  alias Neoboard.Widgets.Mattermost.AuthenticationError
   alias Neoboard.Widgets.Mattermost.Fetcher
 
   def start_link do
     {:ok, pid} = GenServer.start_link(__MODULE__, :ok)
-    send(pid, :tick)
-    :timer.send_interval(config[:every], pid, :tick)
+    send(pid, :login)
```

Comment:
Is it ok to send message to the GenServer even `init` has been called?
IMHO, it's safer to use `send(pid, :login)` in `init(:ok)`.